CONTRIBUTING.md

```diff
@@ -253,33 +253,42 @@ The release process is automated with these steps:
 
 After merging to `main`, a maintainer will:
 
-- View all modules and their current versions:
+- Check out the merge commit:
 
   ```shell
-  ./release.sh --list
+  git checkout MERGE_COMMIT_ID
   ```
 
-- Determine the next version number based on changes:
+- Create annotated tags for each module that was changed:
 
-  - **Patch version** (1.2.3 → 1.2.4): Bug fixes
-  - **Minor version** (1.2.3 → 1.3.0): New features, adding inputs, deprecating inputs
-  - **Major version** (1.2.3 → 2.0.0): Breaking changes (removing inputs, changing input types)
+  ```shell
+  git tag -a "release/$namespace/$module/v$version" -m "Release $namespace/$module v$version"
+  ```
 
-- Create and push an annotated tag:
+- Push the tags to origin:
 
   ```shell
-  # Fetch latest changes
-  git fetch origin
-  
-  # Create and push tag
-  ./release.sh module-name 1.2.3 --push
+  git push origin release/$namespace/$module/v$version
   ```
 
-  The tag format will be: `release/module-name/v1.2.3`
+For example, to release version 1.0.14 of the coder/aider module:
+
+```shell
+git tag -a "release/coder/aider/v1.0.14" -m "Release coder/aider v1.0.14"
+git push origin release/coder/aider/v1.0.14
+```
+
+### Version Numbers
+
+Version numbers should follow semantic versioning:
+
+- **Patch version** (1.2.3 → 1.2.4): Bug fixes
+- **Minor version** (1.2.3 → 1.3.0): New features, adding inputs, deprecating inputs
+- **Major version** (1.2.3 → 2.0.0): Breaking changes (removing inputs, changing input types)
```

Comment:
This should actually be mentioned well before, for the author to put the correct tag in README.md. The maintainer would simply tag whatever tag is in the README while ensuring it's correct as per these rules and ask for changes if not.